src/main/java/com/googleinterns/zoomtube/utils/TranscriptLineUtil.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.googleinterns.zoomtube.data.TranscriptLine;
+import com.googleinterns.zoomtube.servlets.TranscriptServlet;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+/** Provides methods to create TranscriptLine Entities and TranscriptLine objects. */
+public class TranscriptLineUtil {
```

Comment:
Please make this a final class and add a private constructor to indicate this is a util class. Making final prevents inheritance and private constructor allows it not be instantiated.

Reply:
Gotcha. Thanks for letting me know!